.gitmodules

```diff
@@ -0,0 +1,123 @@
+[submodule "cbindgen"]
+	path = cbindgen
+	url = https://github.com/eqrion/cbindgen
```

Comment:
@mkosola I think we need to create still mirrors for each of these. Could you help with those?

Reply:
I think each rust crate mirror should be prefixed with "rust-" similar to this repository. The pattern would then be the same as used by Fedora and it's good to align with some other distro.

Reply:
https://git.sailfishos.org/mirror/rust-cbindgen
https://git.sailfishos.org/mirror/rust-ansi-term
https://git.sailfishos.org/mirror/rust-atty
https://git.sailfishos.org/mirror/rust-autocfg
https://git.sailfishos.org/mirror/rust-bitflags
https://git.sailfishos.org/mirror/rust-cfg-if
https://git.sailfishos.org/mirror/rust-clap
https://git.sailfishos.org/mirror/rust-cloudabi
https://git.sailfishos.org/mirror/rust-getrandom
https://git.sailfishos.org/mirror/rust-hashbrown
https://git.sailfishos.org/mirror/rust-heck
https://git.sailfishos.org/mirror/rust-rusty-hermit
https://git.sailfishos.org/mirror/rust-indexmap
https://git.sailfishos.org/mirror/rust-itoa
https://git.sailfishos.org/mirror/rust-lazy-static.rs
https://git.sailfishos.org/mirror/rust-libc
https://git.sailfishos.org/mirror/rust-parking_lot
https://git.sailfishos.org/mirror/rust-log
https://git.sailfishos.org/mirror/rust-cryptocorrosion
https://git.sailfishos.org/mirror/rust-proc-macro2
https://git.sailfishos.org/mirror/rust-quote
https://git.sailfishos.org/mirror/rust-rand
https://git.sailfishos.org/mirror/rust-syscall
https://git.sailfishos.org/mirror/rust-remove_dir_all
https://git.sailfishos.org/mirror/rust-ryu
https://git.sailfishos.org/mirror/rust-scopeguard
https://git.sailfishos.org/mirror/rust-serde
https://git.sailfishos.org/mirror/rust-json
https://git.sailfishos.org/mirror/rust-serial_test
https://git.sailfishos.org/mirror/rust-smallvec
https://git.sailfishos.org/mirror/rust-strsim-rs
https://git.sailfishos.org/mirror/rust-syn
https://git.sailfishos.org/mirror/rust-tempfile
https://git.sailfishos.org/mirror/rust-textwrap
https://git.sailfishos.org/mirror/rust-toml-rs
https://git.sailfishos.org/mirror/rust-unicode-segmentation
https://git.sailfishos.org/mirror/rust-unicode-width
https://git.sailfishos.org/mirror/rust-unicode-xid
https://git.sailfishos.org/mirror/rust-vec-map
https://git.sailfishos.org/mirror/rust-wasi
https://git.sailfishos.org/mirror/rust-winapi-rs